0meeting-en.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: page
+title: Invitation
+section: meeting
+order: 30
+style: page-doel
+lang: en
+---
+
+## A first meeting
+Have you ever thought that some aspect about your life on campus could be improved by a digital application? Or are you all for transparency? Or maybe you work for a university or college and you want to motivate students to participate in their educational institution?
+
+Join us on **February 15th, at 5 pm near Brussels central station**. On this first meeting, you can meet other people interested in open data in education in Belgium, as well as people who have experience with open data in general. Together, we would like to figure out which data can be interesting to open up. We would also like to get an overview of all things that are open already.
+
+If you're interested, register at [our eventbrite](https://www.eventbrite.nl/e/open-education-kickoff-meeting-registration-29433969796). And be sure to share this with everyone who could be interested as well.
+
+### Why open data?
+The current generation of students was raised with the internet. Their smartphone is always on. And they want to be able to see and do everything on this small device. It's impossible for educational institutions to provide and maintain all tools that are useful for students. But the data that is necessary to build them is property of the institution and often not publicly accessible. Therefore, an entrepreneurial student or staff member can not just build it themselves.
```

Comment:
`Their smartphone is always on` -> `Their smartphone is always close` or something like that